### ISSUE TITLE
Option to add ignored files to Templater settings

### ISSUE THREAD

**Issue by benjaminwelch** [SilentVoid13/Templater#1396]
**Is your feature request related to a problem? Please describe.**
I use Obsidian to study code and right now I'm studying Ruby on Rails. `erb` in Rails is the same syntax as Templater's syntax and this is causing all kinds of strange rendering.

**Describe the solution you'd like** 
If I could add a file to an "ignore these files" list in the settings, then I could have a some documentation for Rails and other languages that wouldn't conflict with Templater's syntax.

**Describe alternatives you've considered** 
I tried disabling Templater while studying but I often create new files which use Templater and this doesn't really work.

**Additional context** 
With Templater enabled on:
<img width="668" alt="Screenshot 2024-06-14 at 08 55 56" src="https://github.com/SilentVoid13/Templater/assets/610197/a61a0feb-4f51-4223-9b4d-be522b23c721">
With Templater disabled:
<img width="689" alt="Screenshot 2024-06-14 at 08 56 57" src="https://github.com/SilentVoid13/Templater/assets/610197/b969d0e7-c82f-43e1-8c94-42d51facaa70">

**Comment by Zachatoo** [comment]
You can disable syntax highlighting in Templater settings to avoid this issue. Seems easier to manage to me than to have to manage a list of files in settings as you add/remove files with Ruby syntax.

**Comment by benjaminwelch** [comment]
@Zachatoo I appreciate the help. Good solution for the immediate. I do appreciate the syntax highlighting when building templates. Limiting syntax highlighting to the designated templates folder/s I choose would work too.

**Comment by macintacos** [comment]
I agree with @benjaminwelch - I'm trying to build a "repository" page with a bunch of snippets that I reuse in actual templates, and having the syntax enabled is making the formatting all messed up. A nice middle grouund would be what Benjamin said - provide an option to enable syntax highlighting in the folders where templates are stored, and disable elsewhere.
